### ISSUE TITLE
Not support for Official Unity-chan

### ISSUE THREAD

**Issue by shinriyo** [unity3d-jp/UnityChanToonShaderVer2_Project#97]
I used the shader for it( official )
https://unity-chan.com/download/releaseNote.php?id=ssu_urp
But it shows pink in color.
What is the reccomended Shader for it?
<img width="510" alt="Screen Shot 2021-12-24 at 23 53 24" src="https://user-images.githubusercontent.com/396927/147360360-9392274e-625c-43d7-9e95-115b73839022.png">

**Comment by H3idi-X** [comment]
What do you mean?
Will you please provide us more detailed information?

**Comment by rihardn** [comment]
Can you provide more details? Pink colour can also be indicator of missing or unsutable material for specific platform.

**Comment by H3idi-X** [comment]
As far as I’ve checked UnityChan SSU URP with Unity 2019.4.34, I was not able to find shader errors on editor.
You mean current version on GItHub cause errors with UnityChan SSU?

**Comment by shinriyo** [comment]
@rihardn @higeometry 
> Will you please provide us more detailed information?
> Can you provide more details?

Thank you for responding.
I just opened in "ユニティちゃんSunny Side Up(URP版)" by Unity 2021.2.7 on macOS Monterey 21.1.
And, later I imported the shader by dragging the plugin package.
I changed the ユニティちゃんSunny Side Up(URP版)'s material `Toon_DoubleShadeWithFeather`.

What is recommended shader for ユニティちゃんSunny Side Up(URP版)?

I would like to run ユニティちゃんSunny Side Up(URP版) on latest Unity on macOS.

**Comment by shinriyo** [comment]
> As far as I’ve checked UnityChan SSU URP with Unity 2019.4.34

Can I run the project for the latest Unity?

Or can I change recommended settings?
```
The Materials in your Project were created using an older version of the Universal Render Pipeline(URP). Unity must upgrade them to be compatible with your current version of URP.
 Unity will re-import all of the Materials in your project, save the upgraded Materials to disk, and check them out in source control if needed.
Please see the Material upgrade guide in the URP documentation for more information.
```

**Comment by H3idi-X** [comment]
You mean the shaders in included in ユニティちゃんSunny Side Up(URP版) doesn't work on your side?
As we are working remotely and other employees are on vacation already.
Hard for me to test on Monterey for now. I'm supposed to get new Macbook installed Monterey middle of January.
We can't provide certain information so far.

**Comment by shinriyo** [comment]
@higeometry 
> You mean the shaders in included in ユニティちゃんSunny Side Up(URP版) doesn't work on your side?

Yes, I do.
It shows black color on my side.

> As we are working remotely and other employees are on vacation already.

I see. Have a nice Christmas.

**Comment by H3idi-X** [comment]
Hi, I haven't checked with MacOS Monterey, but Big Sur with 2021.2.7f1.
Will you please try [2.3.0](https://github.com/unity3d-jp/UnityChanToonShaderVer2_Project/releases/tag/urp-2.3.0) ?

No need to change the shader on materials. Just replace the Package of UTS.

**Comment by riina** [comment]
Hello. If I may, I would like to add that the shader urp-2.3.0 + Unity 2021.2.7 does appear to work on macOS Monterey:
![image](https://user-images.githubusercontent.com/54872398/147379096-61c5e84b-7883-4c36-8c42-43ba8e5b143b.png)
Additionally, I've personally been using this model (URP SSU, I thought she was called Hydran-chan at some point?) in my own project, Unity 2020.3 + [com.unity.toonshader](https://github.com/Unity-Technologies/com.unity.toonshader) 0.2.2-preview, and evidently Unity 2021.2.7 + com.unity.toonshader 0.4.1-preview also works, as shown here:
![image](https://user-images.githubusercontent.com/54872398/147379169-d1a4fe0d-29e8-422e-b113-881baee180af.png)

**Comment by H3idi-X** [comment]
Hi, @shinriyo. I  received new Mac and tried to reproduced the issue.
Maybe, I understand the situation now.

What you want to say is like the following, right?

When we see the pink materials and trying to replace the shader package or making some trials to deal with such scenes, an InternalErrorShader may be assigned to materials sometimes, as shown in the image bellow.  Once you fell into such situation, need to reassign shaders in materials. You should not have to go through such a tedious process in the first place.

We should update https://unity-chan.com/download/releaseNote.php?id=ssu_urp

![image](https://user-images.githubusercontent.com/4477357/148626297-ac1a26c0-1562-4946-a032-104042ea684a.png)

**Comment by H3idi-X** [comment]
Hi, We've just released [ユニティちゃんSunny Side Up(URP版). 1.0.2](https://unity-chan.com/download/releaseNote.php?id=ssu_urp)
Thanks!

**Comment by H3idi-X** [comment]
Fixed in 1.0.2.
